test/utils.jl

```diff
@@ -1,5 +1,16 @@
 using MacroTools: isdef, flatten, striplines, @qq
 
+import StableRNGs
+import Random
+
+@testset "animals" begin
+    @test MacroTools.animals isa Vector{Symbol}
+    @test length(MacroTools.animals) == 214
+    @test allunique(MacroTools.animals)
+    @test all(islowercase ∘ first ∘ string, MacroTools.animals)
+    @test MacroTools.animals == Random.shuffle!(StableRNGs.StableRNG(1234), sort(MacroTools.animals))
```

Comment:
Is that the only reason for the test random dependency? If so I would just take it out, along with StableRNG. I don't see why anyone would care about this particular test, and the less code the better...